examples/Atlas/+atlasParams/Base.m

```diff
@@ -38,7 +38,7 @@
                               'w_qdd', 1e-3 * ones(r.getNumVelocities(), 1),...
                               'integrator', struct('gains', zeros(r.getNumPositions(), 1),...
                                                    'clamps', zeros(r.getNumPositions(), 1),...
-                                                   'eta', 0.005),...
+                                                   'eta', 0.0),...
                               'qdd_bounds', struct('min', -100*ones(r.getNumVelocities(), 1),...
```

Comment:
There was an error in constructQPDataPointerMex in which `eta` was not being parsed (and was thus probabilistically zero). We're now correctly parsing `eta`, so by setting it to zero in the params, we match the prior behavior